### DISCUSSION TITLE
Incompatible architecture problem

### DISCUSSION THREAD

**Discussion by UGAQun** [Lightning-AI/pytorch-lightning#13671] [code help: CV]
I installed pytorch-lightening using conda on M1 Mac Studio. When I tried `import pytorch-lightening as pl`, I got the following error:
OSError: mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64').

**Comment by akihironitta** [comment]
@UGAQun Could you paste the full command you ran (`conda install ...`) and the full traceback? The error should suggest which package you have installed is built for x86_64.

**Comment by UGAQun** [comment]
Hi Akihiro,

Sorry for the late reply. I was busy with something else in the past several months. 
I’m still struggling with installing lightening on my Mac Studio. I followed the instruction on your website and it told me that my environment might be corrupted. So I tried a couple of times to recreate a new env and reinstall everything, which didn’t work. 
Please see attached the full traceback.

I look forward to your response.
Thank you so much!

Best,
Qun

￼

**Reply by akihironitta** [reply]
I believe the error is not specific to Lightning. I hope some of these answers will work: https://stackoverflow.com/questions/71882029/mach-o-file-but-is-an-incompatible-architecture-have-arm64-need-x86-64-i

**Comment by UGAQun** [comment]
Thank you very much! I’ll try it.

Best,
Qun
